test/simple/setMax.js

```diff
@@ -157,5 +157,29 @@ module.exports = simpleEvents({
 
     test.ok(!emitter._events[type].warned, 'should not have been set');
     test.done();
+  },
+
+  'should use process.emitWarning if available' : function (test) {
+
+    // Don't run this test in the browser
+    if(typeof require === 'undefined') {
+      test.done();
+      return;
+    }
+
+    test.expect(2);
+
+    var emitter = new EventEmitter2;
+
+    process.on('warning', function(warning) {
```

Comment:
I think you can test to see if process.emitWarning is defined, and if it isn't either skip or pass the test